SubEthaEdit-Mac/Source/SyntaxHighlighter.h

```diff
@@ -30,9 +30,6 @@ extern NSString * const kSyntaxHighlightingTypeString;
 
 @interface SyntaxHighlighter : NSObject {
     SyntaxDefinition *_syntaxDefinition;
-    // Unused
-    // NSMutableArray *I_parseStack;
-	// NSLock *I_stringLock;
 }
```

Comment:
I took the liberty to remove those alltogether. They are a reminant of a failed multithreading approach that has no more value. Also the whole class might benefit from cleaning up soon.